lib/deck.rb

```diff
@@ -0,0 +1,36 @@
+class Deck
+  attr_reader :cards
+
+  def initialize(cards = [])
+    @cards = cards
+  end
+
+  def rank_of_card_at(position)
+    @cards[position].rank
+  end
+
+  def high_ranking_cards
+    @cards.select do |card|
+      card.rank >= 11
+    end
+  end
+
+  def percent_high_ranking
+    # high_cards = 0.0
+    # @cards.each do |card|
+    #   if card.rank >= 11
+    #     high_cards += 1
+    #   end
+    # end
+    # ((high_cards / @cards.length()) * 100).round(2)
+    ((self.high_ranking_cards.length().to_f / @cards.length()) * 100).round(2)
```

Comment:
Great job refactoring this method to be more dry by using `high_ranking_cards` 🎉